0000-pre-shared-key-v2.md

```diff
@@ -0,0 +1,273 @@
+
+### Chunked Payments
+
+PSKv2 is designed to support both "chunked" and "streaming" payments. Both involve sending many small payments to pay for some interaction. "Chunked Payments" refers to splitting one larger payment into smaller chunks, for example to handle paths with low Maximum Payment Sizes (MPS). "Streaming Payments" refers to payments sent in exchange for an ongoing service, for example paying for a movie stream 1 Mb at a time.
+
+Interledger connectors set their Maximum Payment Size (MPS) based on their available liquidity, the "bandwidth" they allocate to each of their peers, and the risk they are willing to take on each payment. Analogous to the Internet's Path Maximum Transmission Unit (MTU), an Interledger's path MPS is the minimum MPS of the connectors involved.
+
+If senders want to use a path with an MPS lower than their payment amount, they need to split their payment into chunks. PSKv2 includes a Payment ID and Sequence number to enable receivers to associate incoming payment chunks properly.
+
+**Possibility for Incomplete Payments:** If the path between the sender and receiver runs out of liquidity while a chunked payment is in progress, it could be impossible for the payment to complete fully. This should be a rare case, analogous to Internet Service Providers getting cut off because they fail to pay their electricity bills. However, in such cases, the receiver may need to refund the partial payment by sending another payment back and either the sender or receiver would incur the transaction costs.
+
+**Fluctuating Exchange Rates:** The rate from the sender to the receiver may change due to legitimate market movements during the course of a payment. Senders SHOULD monitor the rate changes and they may not want to send payments that are significantly larger than their bandwidth (this could be compared to how downloading a 4K movie over a dial-up connection is possible but would not make for a good experience).
+
+**Malicious Connectors:** Exchange rates can also change for illegitimate reasons, such as connectors driving up the price mid-payment. Ultimately, the main force keeping connectors' rates low is competition. If there is only one path, the connectors do not need to do anything sneaky to drive up the rate. If a sender has multiple options for the connector they send through, they may want to use [Multi-Path Chunked Payments](#multi-path-chunked-payments) to automatically switch paths if one becomes more expensive.
```

Comment:
The point is that they don't need to do anything sneaky. They can simply charge a higher rate up front, since there's nowhere else to go. Sending a chunked payment is no worse than sending a single payment in this case.

Reply:
If such a connector sets a higher rate up front, a customer could simply not send the transfer at all. By providing a cheaper rate initially and increasing the rate half-way through the sender is trapped: Either he finishes the payment at the higher rate or he asks the receiver for a refund, for which the connector would also apply the higher rate. Either way, the sender is trapped.

I propose to drop the sentence _[...] the connectors do not need to do anything sneaky to drive up the rate [...]_.